src/viztracer/modules/vcompressor/vc_dump.c

```diff
@@ -146,6 +147,57 @@ int dump_parsed_trace_events(PyObject* trace_events, FILE* fptr)
         }
     }
 
+    // Iterate through instant events
+    ppos = 0;
+    PyObject* dumps_func = NULL;
+    PyObject* dumps_func_args = PyTuple_New(1);
+    PyObject* args_object = NULL;
+    dumps_func = PyObject_GetAttrString(json_module, "dumps");
+    if (!dumps_func) {
+        goto clean_exit;
+    }
+
+    while (PyDict_Next(instant_events, &ppos, &key, &value)) {
+        // head
+        fputc(VC_HEADER_INSTANT_EVENT, fptr);
```

Comment:
This is similar, we want to keep all the variable declarations on the top of the code block, so `fputc` should be below all the declarations.